src/nvim/getchar.c

```diff
@@ -2382,7 +2400,7 @@ static int vgetorpeek(bool advance)
   --vgetc_busy;
 
   return c;
-}
+} // NOLINT(readability/fn_size)
```

Comment:
linter expects 2 spaces
```suggestion
}  // NOLINT(readability/fn_size)
```

Reply:
Should I clean up the commit history, or leave it as is?